src/style/flatpickr.styl

```diff
@@ -62,6 +62,7 @@
     animation none
     direction ltr
     border 0
+    display none
```

Comment:
No-go. This breaks opening animations for all browsers

Reply:
In which way? I did not see any problem with the default datepicker variant. Using an animation with an early keyframe or an early reset to the needed value in the script to solve this. But so far the high top value and invisible datepicker produces double scrollbars in IE11 and Edge.

Reply:
@DanielRuf noted